en/debug/sensor_uorb_topic_debugging.md

```diff
@@ -0,0 +1,17 @@
+# Sensor/Topic Debugging using the Listener Command
+
+The uORB is an asynchronous `publish()` / `subscribe()` messaging API used for
+inter-thread/inter-process communication. The `listener` command can be used from the *QGroundControl MAVLink Console* to inspect topic (message) values, including the current values published by sensors.
```

Comment:
I'd like to discuss it as a separate topic/offline. I'll move up the note and merge for this case.